kamailio/default.cfg

```diff
@@ -797,24 +804,13 @@ event_route[kazoo:mod-init]
 {
     #!ifdef PRESENCE_ROLE
 
-### use this simple form of binding a listener
-### kazoo_subscribe("dialoginfo", "direct", "BLF-QUEUE-MY_HOSTNAME", "BLF-MY_HOSTNAME");
-###
-### or unleash the power of rabbit to kazoo-blf
-###
-### 'no_ack' : 1 => needs ack,
-### 'wait_for_consumer_ack'
-###    : 1 => when it receives, it processses on the AMQP Worker ad after that it confirms
-###    : 0 => when it receives, it acks then processes in the AMQP Worker
-###    only works if no_ack : 0
-###
-### Rabbit Policy for ha-mode
-###  pattern : ^BLF
-###  definition : ha-mode: all
-###
-###
-
-    $var(payload) = '{ "exchange" : "dialoginfo" , "type" : "direct", "queue" : "BLF-QUEUE-MY_HOSTNAME", "routing" : "BLF-MY_HOSTNAME", "auto_delete" : 0, "durable" : 1, "no_ack" : 0, "wait_for_consumer_ack" : 1 }';
+    $var(payload) = "{ 'exchange' : 'presence' , 'queue' : 'presence-dialog-MY_HOSTNAME', 'exclusive' : 0 ,'type' : 'topic', 'routing' : 'dialog.*.*', 'federate' : 1 }";
```

Comment:
@lazedo what are the defaults for:
* auto_delete
* durable
* no_ack
* wait_for_consumer_ack

?